compiler/ballerina-parser/src/main/java/io/ballerina/compiler/syntax/tree/NodeParser.java

```diff
@@ -147,4 +147,15 @@ public static ModulePartNode parseModulePart(String text) {
         BallerinaParser parser = ParserFactory.getParser(text);
         return parser.parse().createUnlinkedFacade();
     }
+
+    /**
+     * Parses the input as an intermediate clause.
+     *
+     * @param text the input
```

Comment:
Let's add documentation for `allowActions` as well

Reply:
Updated